Realtime: added warning about port 53 URLs (#21770)
cloudflare/cloudflare-docs: src/content/docs/realtime/turn/generate-credentials.mdx

}
```

> Note: the list of URLs returned contains URLs with the primary and alternate ports. The alternate port 53 is known to be blocked by web browsers. So this TURN URL will time out if used in browsers. If you are using trickle ICE this is not going to cause problems. But if you wait for ICE gathering to finish before passing on the ICE candidates you might want to filter out the alternate URL with port 53 to avoid waiting for it to time out.

Reviewer: ```suggestion
:::note
The list of returned URLs contains URLs with the primary and alternate ports. The alternate port 53 is known to be blocked by web browsers, and the TURN URL will time out if used in browsers. If you are using trickle ICE, this will not cause issues, but you should let ICE gathering finish before passing.
:::
```
